recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
 {% set name = "openssl" %}
-{% set version = "3.0.14" %}
+{% set version = "3.0.15" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
-  sha256: eeca035d4dd4e84fc25846d952da6297484afa0650a6f84c682e39df3a4123ca
+#  url: https://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz  # TODO: Revert to this link once the package is on the website. https://openssl-library.org/source/old/3.0/index.html
```

Comment:
```suggestion
```
This page: https://openssl-library.org/source/index.html states that the master sources are maintained in their git repo. All links now point to the git release. The new link seems good to me.